tests/newfeatures/env-deploy.feature

```diff
@@ -9,7 +9,7 @@ Feature: Site Deployment
 
   @vcr site_deploy
   Scenario: Deploy dev to test
-    When I run "terminus env:deploy test --site=[[test_site_name]] --sync-content --note='Deploy test'"
+    When I run "terminus env:deploy [[test_site_name]].test 'Deploy log note' --sync-content"
```

Comment:
@ari-gold: Is the deploy log note optional? If it is I think I would keep the `--note|-n` flag so that the position of optional arguments is not significant... it could be specified before or after `--sync-content` which is also optional. If `--note` is **not** optional then I'm +1 on this arg layout.